aapcs64/aapcs64.rst

```diff
@@ -2026,6 +2026,21 @@ size\_t, ptrdiff\_t
 
 See `Arm C and C++ language mappings`_.
 
+Soft-float variant
```

Comment:
I think this document and aapcs32 hasn't put variant in the title. Presumably inheriting it from "The standard variants". Perhaps just "Soft-float"

Reply:
Done